aptos-move/framework/aptos-framework/tests/deflation_token.move

```diff
@@ -0,0 +1,85 @@
+#[test_only]
```

Comment:
wondering, do we put these example tests here, or in aptos-move/move-examples? 

do these tests get put into the framework package and deployed to a network?

Reply:
My understanding is test_only modules are not included? Also I wanted to have some modules here just serve as the test cases as well. I think we need more polished examples in the move-examples